WordPress/Classes/ViewRelated/NUX/Signin2FAViewController.swift

```diff
@@ -75,6 +76,17 @@ import WordPressShared
     // MARK: Configuration Methods
 
 
+    /// Assigns localized strings to various UIControl defined in the storyboard.
+    ///
+    func localizeControls() {
+        verificationCodeField.placeholder = NSLocalizedString("Verification code", comment: "two factor code placeholder")
+
+        let submitButtonTitle = NSLocalizedString("VERIFY", comment: "Title of a button. The text should be uppercase.").localizedUppercaseString
+        submitButton.setTitle(submitButtonTitle, forState: .Normal)
+        submitButton.setTitle(submitButtonTitle, forState: .Highlighted)
```

Comment:
I took a look around and it seems we're doing this all over the code, so I'll pass this one, if only for consistency.

That said, I think it's a bad idea. It feels like a case of [programming by coincidence](https://pragprog.com/the-pragmatic-programmer/extracts/coincidence) and I think we should stop doing that.

If there's a bug where the highlighted title is not being properly updated we should find the root cause, not add theoretically unnecessary code just because it seems to work that way. If there's an actual UIKit bug, we should first file a radar, then add a documented workaround using a helper method to "set title for all states".